lib/routes/sms.js

```diff
@@ -98,6 +101,53 @@ module.exports = (log, isA, error, config, customs, sms) => {
           return {}
         }
       }
+    },
+    {
+      method: 'GET',
+      path: '/sms/status',
+      config: {
+        auth: {
+          strategy: 'sessionToken'
+        }
+      },
+      handler (request, reply) {
+        log.begin('sms.status', request)
+
+        let isOk
+
+        return getStatus()
+          .then(getLocation)
+          .then(createResponse)
+          .then(reply, reply)
+
+        function getStatus () {
+          return sms.status()
+            .then(status => isOk = status.isOk)
+            .catch(err => {
+              log.error({ op: 'sms.status', err: err })
+              throw error.unexpectedError()
+            })
+        }
+
+        function getLocation (getLocation) {
+          if (isOk) {
+            return getGeoData(request.app.clientAddress)
+              .then(result => {
+                if (! REGIONS.test(result.location.countryCode)) {
```

Comment:
No idea but it seems like the kind of thing we should treat as an error if it's missing.